pkg/doc/verifiable/testdata/context/bss2020.jsonld

```diff
@@ -0,0 +1,97 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "id": "@id",
+    "type": "@type",
+    "ldssk": "https://w3c-ccg.github.io/ldp-bbs2020/context/v1#",
+    "BbsBlsSignature2020": {
+      "@id": "https://w3c-ccg.github.io/ldp-bbs2020/context/v1#BbsBlsSignature2020",
+      "@context": {
+        "@version": 1.1,
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "sec": "https://w3id.org/security#",
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+        "challenge": "sec:challenge",
+        "created": {
+          "@id": "http://purl.org/dc/terms/created",
+          "@type": "xsd:dateTime"
+        },
+        "domain": "sec:domain",
+        "proofValue": "sec:proofValue",
+        "nonce": "sec:nonce",
+        "proofPurpose": {
+          "@id": "sec:proofPurpose",
+          "@type": "@vocab",
+          "@context": {
+            "@version": 1.1,
+            "@protected": true,
+            "id": "@id",
+            "type": "@type",
+            "sec": "https://w3id.org/security#",
+            "assertionMethod": {
+              "@id": "sec:assertionMethod",
+              "@type": "@id",
+              "@container": "@set"
+            },
+            "authentication": {
+              "@id": "sec:authenticationMethod",
+              "@type": "@id",
+              "@container": "@set"
+            }
+          }
+        },
+        "verificationMethod": {
+          "@id": "sec:verificationMethod",
+          "@type": "@id"
+        }
+      }
+    },
+    "BbsBlsSignatureProof2020": {
```

Comment:
Are we using `BbsBlsSignatureProof2020` json-ld type ? Or is it just here as we copied the context file entirely from the source ?

Reply:
Yes, we start to support it. But in the tests, we do not want to rely on the network availability of the contexts, and that's why we cache `BbsBlsSignatureProof2020 ` context definition (referenced by `https://w3c-ccg.github.io/ldp-bbs2020/context/v1`). If you check `pkg/doc/verifiable/testdata/context/`, you can see much more cached contexts there... The contexts caching also speed ups the tests - I recall there was a speedup of `verifiable` tests run from about 2 minutes to several seconds after we started to use the context cache in the tests.

Reply:
perhaps this file should be prefixed (or suffixed) with cache to indicate this is caching for testdata